packages/screenshots/Classes/Configuration/Configuration.php

```diff
@@ -221,7 +221,34 @@ public function createBasicConfig(): void
                 ],
                 'Examples' => [
                     'screenshots' => [
-                        [
+                        "_default"=> [
+                            ["action"=> "resizeWindow", "width"=> 640,"height"=> 640],
+                            ["action"=> "setScreenshotsDocumentationPath", "path"=> "Documentation"],
+                            ["action"=> "setScreenshotsImagePath","path"=> "Images/AutomaticScreenshots"],
+                            ["action"=> "setScreenshotsRstPath","path"=> "Images/AutomaticScreenshots"],
+                            ["action"=> "setCodeSnippetsSourcePath","path"=> "typo3conf/ext/examples"],
+                            ["action"=> "setCodeSnippetsTargetPath", "path"=> "CodeSnippets"]
+                        ],
+                        "frontend" => [
```

Comment:
Keep prefix "actionsIdentifier.." for all identifiers in the MyManual. See other identifiers.

Reply:
Whats the point of using such long and complicated names?